enhancements/console/guided-tours.md

```diff
@@ -0,0 +1,144 @@
+* ...
+
+In order to achieve the desired outcome user needs to create a set of resources,
+that are dependent on each other.
+
+For Guided Tours we need a mechanism for their creation and publishment.
+
+## Motivation
+
+Help users with their understanding the principles of their workflows by
+guiding them though the necessary steps.
+
+### Goals
+
+1. Provide a mechanism to display user guides for various workflows.
+2. Make users understand what are the steps needed to achieve their goals.
+3. Provide API for writing Guided Tours.
+4. Provide new CRD format for writing the Guided Tours.
+5. Have a storage for the Guided Tours.
+
+## Proposal
+
+* Introduce CRD format that will be used for writing Guided Tours.
+* Introduce default storage for Guided Tours.
+  
+### User Stories
+
+#### Story 1
+
+As an administrator of an OpenShift cluster, I need a guide to walk me through how to install OpenShift Serverless modules (Serving and Eventing) in a cluster.
+
+#### Story 2
+
+As an administrator of an OpenShift cluster, I need a guide to walk me through how to update an OpenShift cluster.
+
+#### Story 3
+
+As a developer, I need a guide to walk me through how to deploy an existing application as a serverless workload.
+
+#### Story 4
+
+As a operator creator I want to provide operator consumers with a guide on how to install and user the my operator.
+
+### Implementation Details
+
+1. In order to provide mechanism to discribe a Guided Tour, new CRD named `GuidedTours` will be created.
+2. A new `openshift/guided-tours` repository will be created which will contain all supported Guided Tours.
+3. `console-operator` will import all the existing Guided Tours CRs from the `openshift/guided-tours` repository into the `/manifest` directory, so that the CVO can:
+   * create them if the CR doesn't exists.
+   * update the CRs upon the cluster update (since CVO is doing `apply`).
+4. `console-operator` will check for older `GuidedTours` CR versions and remove those that doesn't match the cluster major version, which indicates that new version is not available.
+5. Steps in the Guided Tours will support basic markdown thats already in use in the OpenShift's Console. 
+6. All Guided Tours will be listed available in a separate page that will be accessible from Help Menu.
+   ![help-menu](https://raw.githubusercontent.com/jhadvig/images/master/help-menu.png)
+7. For different areas of interest, the links for Guided Tour will need to be handles separately:
+   * Operators - CVS of the operator that has a Guided Tours published will contain `console.openshift.io/guided-tour: <guided-tour-name>` annotation. Upon the operator installation an "Guided Tour" link will appear next to the `Install` button.
+   ![operators](https://raw.githubusercontent.com/jhadvig/images/master/operators.png)
+   * ...TBD
+
+
+GuidedTours CR for [Explore Serverless](https://marvelapp.com/236ge4ig/screen/69908905):
+```
+apiVersion: console.openshift.io/v1
+kind: GuidedTours
```

Comment:
Kind should be singular, `GuidedTour` (although I know you're working on changing the name anyway)